packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -1861,7 +1861,7 @@ describe('/account/login', () => {
         );
         assert.equal(
           response.verificationReason,
-          'password-change',
+          'change_password',
           'verificationReason is password-change'
         );
       });
```

Comment:
I think we need to add a test for when config.forcePasswordChange is the reason for verification instead of lockedAt.

Reply:
Added